code/hub.dm

```diff
@@ -1,16 +0,0 @@
-#define HUB_ENABLED
-
-/world
-	hub = "Exadv1.spacestation13"
-	name = "Space Station 13"
-#ifdef HUB_ENABLED
```

Comment:
This needs to be maintained in the new version.

Reply:
For the reason of...? It's literally defined at the top of the file, doing nothing

Reply:
Yeah, you are right, I remembered the old system where the hub password was not set by default